src/main/steps/children/other-adoption-agency/content.ts

```diff
@@ -0,0 +1,63 @@
+import { YesOrNo } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+
+const en = () => ({
+  section: 'Your adoption agency or local authority details',
+  title: 'Was there another adoption agency or local authority involved in placing the child?',
```

Comment:
Please make it label
```suggestion
  label: 'Was there another adoption agency or local authority involved in placing the child?',
```